lib/services/downloads_service.dart

```diff
@@ -561,6 +571,29 @@ class DownloadsService {
             centerText: true));
   }
 
+  Future<bool> canDeleteFromServer({required String itemId, alreadyChecked = false}) async {
```

Comment:
I feel like this function could also feel at home in the `jellyfin_api_helper`, after all that's an abstraction of the actual `jellyfin_api`, and it doesn't interact with the actual download system.

Reply:
Fair, moved it to the api helper (will push later)